Library/Model/SPIPayAtTable.h

```diff
@@ -100,15 +100,16 @@ typedef NS_ENUM(NSUInteger, SPIPaymentType) {
 
 @interface SPIPayAtTableConfig : NSObject
 
+@property (nonatomic) BOOL payAtTableEnabled;
 @property (nonatomic) BOOL operatorIdEnabled;
 @property (nonatomic) BOOL splitByAmountEnabled;
 @property (nonatomic) BOOL equalSplitEnabled;
 @property (nonatomic) BOOL tippingEnabled;
 @property (nonatomic) BOOL summaryReportEnabled;
-@property (nonatomic, readonly, copy) NSString *labelPayButton;
-@property (nonatomic, readonly, copy) NSString *labelOperatorId;
-@property (nonatomic, readonly, copy) NSString *labelTableId;
-@property (nonatomic, readonly, copy) NSArray<NSString *> *allowedOperatorIds;
+@property (nonatomic) NSString *labelPayButton;
```

Comment:
Reason for change?

Reply:
They couldn't be set a value when they were read-only

Reply:
I was more talking about the removal of `copy`, this will impact the behaviour.

Reply:
ok I got it. Should it be like that `@property (nonatomic, copy) NSString *labelPayButton;`?

Reply:
Copy is usually a safe bet for things like strings / arrays where performance isn't a massive deal.

Reply:
Added like that @property (nonatomic, copy) NSString *labelPayButton;